numpy/core/tests/test_scalarprint.py

```diff
@@ -45,6 +46,20 @@ def check(v):
         check(1e15)
         check(1e16)
 
+        # gh-10753
```

Comment:
I think this should be its own test function - it doesn't really test the same thing.